### DISCUSSION TITLE
notifica che segnala il cambiamento del meteo

### DISCUSSION THREAD

**Discussion by pifma** [indomus/forum#2959] [HUB: Home Assistant]
Vorrei inviare una notifica con la percentuale di nuvolosità ma non riesco a capire cosa devo mettere nelle parentesi graffe
```weather.hac_home_assistant_control.cloud_coverage```
Quale è la sintassi corretta?

**Comment by snakuzzo** [comment]
Ti consiglio vivamente di studiare come funzionano i template...

https://indomus.it/formazione/esercizi-di-domotica-su-home-assistant-1-creare-dei-sensori-template/

**Comment by pifma** [comment]
risolto grazie
{{ states('yyyy.xxxx') }}
